src/pages/Game/components/PlanetIcon/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import './styles.scss'
+import saturn from '../../../../assets/images/saturn.png'
+import earth from '../../../../assets/images/earth.png'
+
+const PlanetIcon = ({player}) => {
+  /*colocar estilo en styles*/
+  return(
+  player === 'X' ? <img src={saturn} alt="X" width="90" height="90" /> : player === 'O' ? <img src={earth} alt="O" width="90" height="90" /> : null
```

Comment:
Just a comment, but avoid using shorthand IF when you need to nest or express an `else` statement. Use a regular IF structure instead.